src/cclib/io/moldenwriter.py

```diff
@@ -78,6 +109,32 @@ def _scfconv_from_ccdata(self):
 
         return lines
 
+    def _rearrange_mocoeffs(self, mocoeffs):
```

Comment:
The ordering issue occurs in only `f` orbitals. `d`, `g` and others were fine. (I had exhaustively checked the files in `data` folder.)